contents/blog/2022-review.md

```diff
@@ -0,0 +1,78 @@
+
+At one point, I was getting the little things done, but not the big ones. Tim suggested I hire Kendal (Exec Assistant). She helped solve this.
+
+At another stage, Tim and I realised we were managing the wrong teams because the game had changed. I was good at early sales, but he's the right person to help us scale things there. Likewise in product, we want to go from good to great and that'll take more design skill which I can help with. We swapped a bunch of priorities around with each other, and - so far - it feels much better!
+
+### A crappy family situation got much better
+
+My (then) 1 year old daughter got diagnosed with bilateral retinoblastoma (a type of cancer) in 2021. She lost an eye as a result, has her vision permanently damaged in the other, and had to go through chemo. We were delighted to wrap up treatment in the Spring 2022. We were lucky that it went perfectly... she was cured, and now we're in regular monitoring mode. When she got diagnosed, I chose to focus on revenue over fundraising (instead of trying to both whilst taking care of her), and now a year and a half on, I'm very glad that's what we picked.
+
+### We are shipping stuff
+
+We made too many improvements to list on user facing functionality, and infra / ingestion based functionality. It feels like we can make new things appear, although Q1 will be a big test - it'll be one of our most ambitious quarters.
+
+Other teams also get their work out the door quickly, unimpeded by bureaucracy / excessive planning. We've had a ton of marketing content appearing, a smooth time with our People and Ops team, and a growth team that can't stop producing winning variants!
+
+### I'm really glad we're not doing outbound / went for a growth team instead
+
+It is always tempting to hire an outbound sales team. After all, we can sell the product and retain users, to a tightly defined audience.
+
+However, it just feels like it'd be very hard to do outbound sales to developers. I can't think of a worse group to target this way (this is coming from a place of experience - I've run outbound sales teams before).
+
+Instead, we set up a growth team. We've had dramatic results from doing this - such as 25% improvements in our activation flow from single experiments. Yep, one experiment has led to a 25% increase in our monthly growth, forever. That's why we have such a low CAC payback period...
+
+If we _had_ hired a sales team, I guarantee we'd be focussing on how to send more cold emails more effectively versus the more leveraged work of improving our onboarding, activation and billing flows.
+
+### We can take pride in not focussing on number of staff / investors
+
+Lots of companies raised very hyped funding rounds in 2021 - we didn't. We are confident we made the right call not to do this, but we felt that we were going against the grain. It was really cool to talk about this stuff openly with our team, and to see how quickly we started taking pride in building a real company.
```

Comment:
I would maybe add in some way that we could have done this but didn't? (Loads of companies of course didn't raise a very hyped round - because they were never offered one!)